MathGame.JsPeanut/MathGame.JsPeanut/Program.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System;
+
+class Program
+{
+
+    public static void Main(string[] args)
+    {
+        List<Operation> previousGames = new List<Operation>();
+
+        bool doUserStillWantToPlay = true;
+
+        Console.WriteLine("Hello there! Welcome to this math game.");
+        Console.WriteLine("Available commands: -Score -Menu");
+        Console.WriteLine("Do you want to play?");
+        Console.WriteLine("Type 'yes' if so. If not, type 'exit'.");
+
+        while (doUserStillWantToPlay)
+        {
+            switch (Console.ReadLine())
+            {
+                case "yes":
+                    StartGame();
+                    break;
+                case "exit":
+                    doUserStillWantToPlay = false;
+                    break;
+            }
+        }
+
+        void StartGame()
+        {
+            Random rnd = new Random();
```

Comment:
We can shorten this to the following:
```suggestion
            Random rnd = new();
```